cmd/cvdr/main.go

```diff
@@ -24,24 +25,40 @@ import (
 	"github.com/google/cloud-android-orchestration/pkg/client"
 )
 
-const configPathVar = "CVDR_CONFIG_PATH"
+const (
+	envVarSystemConfigPath = "CVDR_SYSTEM_CONFIG_PATH"
+	// User config values overrieds system config values.
+	envVarUserConfigPath = "CVDR_USER_CONFIG_PATH"
+)
 
-func readConfig(config *cli.Config) error {
-	configPath := os.Getenv(configPathVar)
-	if configPath == "" {
-		// No config file provided
-		return nil
+func loadInitialConfig() (*cli.Config, error) {
```

Comment:
I think this function could be simplified:

- Refactor `cli.LoadConfigFile` or create a new function with this signature: `func LoadConfigFile(filePath string, configObj *Config) error`
- Reimplement this function like this:
```
config := cli.DefaultConfig();
if path, ok := os.LookupEnv(envVarSystemConfigPath); ok {
  // ...
  err := cli.LoadConfigFile(path, config)
  // ...
}

if path, ok := os.LookupEnv(envVarUserConfigPath); ok {
  // ...
  // TODO if file doesn't exist generate it
  // ...
  err := cli.LoadConfig(path, config)
  // ...
}
return config
```

Reply:
true, done.